package.json

```diff
@@ -46,7 +46,7 @@
   },
   "devDependencies": {
     "axe-core": "^4.5.2",
-    "cypress": "^12.17.1 ",
+    "cypress": "^12.17.1",
```

Comment:
Remove the "^" here - if another version of Cypress is installed there's a version mis-match with the Docker-e2e Dockerfile and it causes issues.

Reply:
Cool, just made changes to remove it